### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by paulacarneiro**
Why:
- There's currently no linting setup

How:
- Add eslint and pettier packages and configs for astro and svelte
- Create config files and scripts
- Format and lint code

### CODE REVIEW

**Comment by zediogoviana** [comment]
Will we have any preview for this?

**Comment by paulacarneiro** [comment]
i'm not sure

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+.DS_Store
+node_modules
+.env
+.env.*
+!.env.example
+
+yarn.lock
```
Comment (zediogoviana) [inline .prettierignore]: Adding an empty line at the end
```suggestion
yarn.lock

```

**Review by zediogoviana — APPROVED** [review]
Looks good! It'd be cool if you could set up a preview with Netlfify/wtv for this

**Review by zediogoviana — APPROVED** [review]

**Pull request merged**